distribution/ddf-common/src/main/resources/etc/org.apache.karaf.command.acl.ssh.cfg

```diff
@@ -0,0 +1,5 @@
+#
+# This configuration file defines the ACLs for commands in the ssh subshell
+#
+
+* = NO_ACCESS
```

Comment:
Does this mean if I locally ssh into an instance that i will not have any of the built in shell commands?

Reply:
It means that all ssh commands are disabled. You can't shell out from karaf.